playground/keycloak/Keycloak.AppHost/Program.cs

```diff
@@ -0,0 +1,13 @@
+var builder = DistributedApplication.CreateBuilder(args);
+
+var keycloak = builder.AddKeycloak("keycloak");
```

Comment:
Enable the data volume here?

Reply:
I can but it complicates the playground. You'll have to also setup a password and you'll have to set it in user secrets.

Does that not complicate the playground?

Reply:
Ah OK that's fine for now then. Once #1151 is resolved we can revisit.

Reply:
@DamianEdwards Enabled the data volume in the last iteration. Seems to be working great!